client/components/skeleton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Center, Skeleton, VStack } from 'native-base';
```

Comment:
Move to ui folder, try to minimize native base usage. We might want to pull the native base skeleton source code into our own ui component

Reply:
i tried a few methods but tamagui does not comes with a built-in Skeleton. so i moved the native-base code into packrat.